spec/parallel_spec.rb

```diff
@@ -188,7 +188,7 @@ def cpus
 
     it 'does not leave processes behind while running' do
       skip if ENV['TRAVIS'] # this randomly fails on travis all the time :(
-      `ruby spec/cases/closes_processes_at_runtime.rb`.should == 'OK'
+      `ruby spec/cases/closes_processes_at_runtime.rb`.gsub(/[^\n]* deprecated; use BigDecimal[^\n]*\n/, '').should == 'OK'
```

Comment:
no need for `[^\n]` since regexes don't jump lines

`"abc\nb".gsub(/a.*/, "") == "\nb"`